include/deal.II/base/mpi_compute_index_owner_internal.h

```diff
@@ -29,21 +29,29 @@ namespace Utilities
     namespace internal
     {
       /**
-       * An internal namespace used for Utilities::MPI::compute_index_owner().
+       * An internal namespace used for Utilities::MPI::compute_index_owner()
+       * and for Utilities::MPI::Partitioner::set_ghost_indices().
        */
       namespace ComputeIndexOwner
       {
+        /**
+         * Dictionary class with basic partitioning in terms of a single
+         * interval of fixed size known to all MPI ranks for two-stage index
+         * lookup.
+         */
         struct Dictionary
         {
           static const unsigned int tag_setup = 11;
 
           std::vector<unsigned int> actually_owning_ranks;
+          std::vector<unsigned int> actually_owning_rank_list;
```

Comment:
Can you add a comment about the difference between the two variables

Reply:
I will add descriptions to all members in this struct.